src/pages/UDT/UDTComp.tsx

```diff
@@ -166,6 +167,20 @@ export const UDTOverviewCard = ({ typeHash, udt }: { typeHash: string; udt: UDT
         className={styles.cardHeader}
         title={!isMobile && cardTitle}
         hash={typeHash}
+        customActions={[
+          isOmigaInscriptionCollection(udt) && udt.mintStatus === MintStatus.RebaseStart ? (
+            <Tooltip placement="top" title={t('udt.view_original')}>
+              <a
```

Comment:
Why not use `Link` here

Reply:
This opens in a new tab, so theoretically there's no difference between `link` and `a`.
After thinking about it, using `link` can be more conducive to subsequent maintenance, so I updated it.